libass/ass_font.c

```diff
@@ -87,12 +237,22 @@ uint32_t ass_font_index_magic(FT_Face face, uint32_t symbol)
     if (!face->charmap)
         return symbol;
 
-    switch (face->charmap->encoding) {
-    case FT_ENCODING_MS_SYMBOL:
-        return 0xF000 | symbol;
-    default:
-        return symbol;
+    if (face->charmap->platform_id == TT_PLATFORM_MICROSOFT) {
```

Comment:
This is a good and innocent change (bug fix), but a nonobvious change nevertheless. Might be worth mentioning in the commit message, at least?